src/scores.ts

```diff
@@ -264,94 +258,106 @@ export function scaleEnabledAttributeScore(score: number) {
 }
 
 function shouldHideCommentDueToAttributeScore(
-  attributeScore: AttributeScore, threshold: number)
-: boolean {
-  if (threshold > BLARING_THRESHOLD) {
-    return false;
+  attributeScore: AttributeScore|null, threshold: number)
+: HideCommentDueToScores|null {
+  if (attributeScore === null || threshold > BLARING_THRESHOLD) {
+    return null;
   }
   const boundedThreshold = Math.max(threshold, LOW_THRESHOLD);
   const scaledScore = scaleEnabledAttributeScore(attributeScore.score);
-  return scaledScore >= boundedThreshold;
+  if (scaledScore < boundedThreshold) {
+    return null;
+  }
+  return {
+    kind: 'hideCommentDueToScores',
+    attribute: attributeScore.attribute,
+    scaledScore: scaledScore,
+  };
 }
 
-function shouldHideCommentDueToLowQuality(
-  scores: AttributeScores, threshold: number)
-: boolean {
+function shouldHideCommentDueToLowQuality(scores: AttributeScores, threshold: number)
```

Comment:
Just for extra clarity, could you add a function level comment that says that when a comment is hidden for low quality, we use the toxicity score? I know it's in the code, but since it isn't entirely obvious, I think it's worth the redundancy.